benchmark_runner/benchmark_operator/benchmark_operator_workloads_operations.py

```diff
@@ -74,8 +74,8 @@ def __init__(self):
         if self._enable_prometheus_snapshot:
             self._snapshot = PrometheusSnapshot(oc=self._oc, artifacts_path=self._run_artifacts_path, verbose=True)
         self._prometheus_metrics_operation = PrometheusMetricsOperation()
-        # update lso_disk_id
-        if self._oc.get_free_disk_id():
+        # Update lso_disk_id only if worker_disk_ids exist
+        if self._environment_variables_dict.get('worker_disk_ids', ''):
             self._lso_disk_id = self._oc.get_free_disk_id()
```

Comment:
You need to make sure that you actually find a disk here -- it's all well and good for the secrets to specify a disk ID, but you need to make sure that you can find one at runtime.

Reply:
add self._oc.get_free_disk_id()

Reply:
What happens if `self._oc_get_free_disk_id()` does not find a suitable disk?

Reply:
OK.  I think it should raise an exception if it doesn't get a free disk ID (this is a real problem), but however you want to handle it.  What happens if, for example, the only remaining disk goes offline?

Reply:
Fixed